src/c/psychrolib.c

```diff
@@ -53,12 +53,18 @@
  * Global constants
  *****************************************************************************************************/
 
-#define R_DA_IP 53.350    // Universal gas constant for dry air (IP version) in ft∙lbf/lb_da/R
-                          // Reference: ASHRAE Handbook - Fundamentals (2017) ch. 1
-#define R_DA_SI 287.042   // Universal gas constant for dry air (SI version) in J/kg_da/K
-                          // Reference: ASHRAE Handbook - Fundamentals (2017) ch. 1
+#define R_DA_IP 53.350            // Universal gas constant for dry air (IP version) in ft∙lbf/lb_da/R
+                                  // Reference: ASHRAE Handbook - Fundamentals (2017) ch. 1
+#define R_DA_SI 287.042           // Universal gas constant for dry air (SI version) in J/kg_da/K
+                                  // Reference: ASHRAE Handbook - Fundamentals (2017) ch. 1
+#define INVALID -99999            // Invalid value
 
-#define INVALID -99999    // Invalid value
+#define MIN_ITER_COUNT 3          // Minimum number of iterations before exiting while loops.
```

Comment:
For me MIN_ITER_COUNT is not needed. It's good to do defensive coding (as is the code with MAX_ITER_COUNT) but in this case it doesn't serve any purpose, if convergence has been reached on the first pass then there is no need to continue iterating.

Reply:
I agree, I have removed MIN_ITER_COUNT from all.